release-notes/RC1-RC2_Upgrade.md

```diff
@@ -0,0 +1,44 @@
+# RC1 to RC2 Upgrade Roadmap
+
+Due to schedule changes for the .NET Core all-up, there was a rename of the tooling from RC2 to RC2 - Preview 1. This rename impacted all of the packages as well as installers for .NET Core and .NET Core tools. The impact of this change for the users is in that all previous versions of .NET Core and any tooling **must be** removed from the machine in order to properly install and use RC2 release.
```

Comment:
Chatted with Z and it makes sense now. "RC2 - Preview 1" is correct for tooling. I was confused until I thought of it as the tooling Preview 1 for RC2.